### ISSUE TITLE
Unnecessary `=""` added

### ISSUE THREAD

**Issue by kevinansfield** [ember-codemods/ember-no-implicit-this-codemod#53]
Before:
```html
<a href="" class="theme-validation-toggle-details" {{action "toggleDetails"}} data-test-toggle-details>
```

After:
```html
<a href="" class="theme-validation-toggle-details"="" {{action "toggleDetails"}}="" data-test-toggle-details>
```

Here's the full original file in case it's needed for debugging:
<details>
    <summary>gh-theme-error-li.hbs</summary>

```handlebars
<a href="" class="theme-validation-toggle-details" {{action "toggleDetails"}} data-test-toggle-details>
    <div class="theme-validation-type-label"></div>
    <div class="flex items-center flex-auto">
        <h4 class="theme-validation-rule-text">
            {{{this.error.rule}}}
        </h4>
        <div class="theme-validation-rule-icon">
            {{#if this.showDetails}}
                {{svg-jar "arrow-down"}}
            {{else}}
                {{svg-jar "arrow-right"}}
            {{/if}}
        </div>
    </div>
</a>

{{#if this.showDetails}}
    <p class="theme-validation-details">
        {{{this.error.details}}}
    </p>
    {{#if this.error.failures}}
        <div class="theme-validation-list">
            <h6>Affected files:</h6>
            <ul>
                {{#each this.error.failures as |failure|}}
                    <li><code>{{failure.ref}}</code>{{#if failure.message}}: {{failure.message}}{{/if}}</li>
                {{/each}}
            </ul>
        </div>
    {{/if}}
{{/if}}
```

</details>

**Comment by Turbo87** [comment]
this may have been caused by a bug in ember-template-recast. I've tried the codemod with your snippet above and `master` seems to return a correct result now.

please let me know if this is still an issue with the new version.

**Comment by Turbo87** [comment]
@kevinansfield this was just fixed in ember-template-recast 3.3.1. please let us know if this is still an issue for you :)
